tripy/tripy/frontend/module/module.py

```diff
@@ -313,12 +323,14 @@ def _iterate_members_of_type(self, typ: T) -> Iterator[Tuple[str, T]]:
         for name, value in vars(self).items():
             if isinstance(value, typ):
                 yield name, value
-            elif isinstance(value, List) and _is_homogeneous_container(value, typ):
+            elif isinstance(value, List):
                 for i, obj in enumerate(value):
-                    yield f"{name}.{i}", obj
-            elif isinstance(value, Dict) and _is_homogeneous_container(value.values(), typ):
+                    if isinstance(obj, typ):
+                        yield f"{name}.{i}", obj
+            elif isinstance(value, Dict):
```

Comment:
Is `_is_homogeneous_container` used anywhere? If not we can delete it

Reply:
deleted